### ISSUE TITLE
Error: Found invalid baseIRI '&ssnx;ssn' for value '&xsd;string'

### ISSUE THREAD

**Issue by thomasdegroot18** [rdfjs/rdfxml-streaming-parser.js#10]
Hello,

We wanted to implement your library, but we ran into a issue with the implmentation. According to the https://www.w3.org/RDF/Validator/rdfval the XML file, below should be parsable, but he library blocked the parsing of the XML. This was due to an error with the Invalid baseIRI '&ssnx;ssn' for value '&xsd;string'. 
I think you missed a case in the library that catches an error if the baseIRI does not match the String.

The minimal XML example:

```<?xml version="1.0"?>
<!DOCTYPE rdf:RDF [
    <!ENTITY ssnx "http://purl.oclc.org/NET/ssnx/" >
    <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#" >
    <!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
    <!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#" >
]>

<rdf:RDF xmlns="&ssnx;ssn#"
     xml:base="&ssnx;ssn"
     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
     xmlns:ssnx="http://purl.oclc.org/NET/ssnx/"
     xmlns:owl="http://www.w3.org/2002/07/owl#"
     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
    <owl:Ontology rdf:about="">
        <rdfs:comment rdf:datatype="&xsd;string">Please report any errors to the Semantic Sensor Network Incubator Activity via the public W3C list public-xg-ssn@w3.org</rdfs:comment>
    </owl:Ontology>
</rdf:RDF>```

**Comment by rubensworks** [comment]
Indeed, that looks like a bug on my end.

Thanks for reporting, I'll look into it soon!

**Comment by rubensworks** [comment]
Fixed in 1.2.1

**Comment by thomasdegroot18** [comment]
Thank you for fixing it so quickly.
